src/Microsoft.ApplicationInsights/Managed/Shared/Extensibility/Implementation/Tracing/HeartbeatProvider.cs

```diff
@@ -26,6 +26,11 @@ internal class HeartbeatProvider : IDisposable, IHeartbeatProvider
         /// </summary>
         public static readonly TimeSpan MinimumHeartbeatInterval = TimeSpan.FromSeconds(30.0);
 
+        /// <summary>
+        /// Value for property indicating 'app insights version' related specifically to heartbeats.
+        /// </summary>
+        private static string sdkVersionPropertyValue = SdkVersionUtils.GetSdkVersion("hbeat:");
```

Comment:
just 'hb' is sufficient to save space.

Reply:
Agreed. Will make the change.

Reply:
Done. Please accept PR Microsoft/ApplicationInsights-Home#214 along with this one.